test-vectors/did_jwk/resolve.json

```diff
@@ -7,10 +7,7 @@
       "output": {
         "@context": "https://w3id.org/did-resolution/v1",
         "didDocument": {
-          "@context": [
-            "https://www.w3.org/ns/did/v1",
-            "https://w3id.org/security/suites/jws-2020/v1"
-          ],
+          "@context": "https://www.w3.org/ns/did/v1",
```

Comment:
forcing @context to be a string, not a list of strings

Reply:
could you force this to be a list of strings? I think that will be the most common case for context (if we should choose to use it)

Reply:
sure, that sounds good! i'll update the original PR to build DIDDocument with a list of string for `@context` now.